### PR TITLE
Dev pm 5340, fixed bouncing ball example

### PR DESCRIPTION
**Pull request by PeterMeisrimelModelon**
load_fmu change removed a deprecation warning.
rtol was undefined

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -4,6 +4,7 @@
       data from the loaded file.
     * Added option "synchronize_simulation" to allow for synchronizing 
       simulation with (scaled) real-time.
+    * Bugfix in fmi20_bouncing_ball_native.py, would trigger a step_event every timestep.
```
Comment (modelonrobinandersson) [inline CHANGELOG:7]: I might be a bit pedantic here but I would rephrase _bugfix_ to something else, such as `Resolved a bug in the example fmi20_bouncing_ball_native where a time step is triggered for every step event.`

File: src/pyfmi/examples/fmi20_bouncing_ball_native.py [hunk 2]
```diff
@@ -104,11 +104,13 @@ def run_demo(with_plots=True):
...
         #Inform the model about an accepted step and check for step events
-        step_event = bouncing_fmu.completed_integrator_step()
+        step_event, terminate = bouncing_fmu.completed_integrator_step()
+        if terminate: 
+            return
         
         #Check for time and state events
         time_event  = abs(time-Tnext) <= 1.e-10
-        state_event = True if True in ((event_ind_new>0.0) != (event_ind>0.0)) else False
+        state_event = True in ((event_ind_new>0.0) != (event_ind>0.0))
```
Comment (modelonrobinandersson) [inline src/pyfmi/examples/fmi20_bouncing_ball_native.py:113]: I might be missing something but isn't it enough to write `state_event = (event_ind_new>0.0) != (event_ind>0.0)`?

**Review by modelonrobinandersson — CHANGES_REQUESTED** [review]
Looks good, just two minor comments

File: CHANGELOG (hunk 1 above)
Comment (PeterMeisrimelModelon) [reply CHANGELOG]: I like that suggestion, changed it to:

`Resolved a bug in the example fmi20_bouncing_ball_native where a step-event is triggered in every timestep.`

File: src/pyfmi/examples/fmi20_bouncing_ball_native.py (hunk 2 above)
Comment (PeterMeisrimelModelon) [reply src/pyfmi/examples/fmi20_bouncing_ball_native.py:113]: In general, no, since event_ind(_new) is an array of all event indicators and this checks if any changed sign.

It would work in this particular example though, since there is only a single event indicator. I'd still keep the general one though.

Comment (modelonrobinandersson) [reply src/pyfmi/examples/fmi20_bouncing_ball_native.py:113]: I see, thank you for the clarification!

**Review by modelonrobinandersson — APPROVED** [review]
Looks great!
